### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by wwzz-max** [ChenLiu-1996/CUTS#8]
Hello, thank you very much for your work. I have a question: with only 56 images in the retinal dataset, will reviewers have doubts about the results obtained from such a small dataset? I am also researching in this direction and am a beginner, so I don't understand much about this. Could you please clarify my doubts?

**Comment by ChenLiu-1996** [comment]
I have shared the brain ventricles dataset with you via email.

Yes what you said is a possible concern. And I would not be surprised if reviewers are unhappy about it. The reason for us using these small datasets are (1) that is all we got from our collaborators and (2) currently our method is training reasonably fast but during inference it is quite slow so we cannot afford running too many images. If you are a beginner in this area, I would recommend starting off with some other baselines, and using large public datasets.

You can start off with DFC, STEGO and SAM which we have code on how to interface with --- and please still cite us if we are being helpful.

**Comment by wwzz-max** [comment]
Thank you for your response. Yes, I am familiar with DFC, SETGO, and SAM. However, as an individual, my computational resources are very limited. I am the only one in our group who chose to research this direction due to my strong interest. When I was struggling without any assistance, I came across your article on CUTS. The approach of CUTS is ingenious, easy to understand, and surprisingly effective. I deeply admire this work. The computational requirements are within my capabilities, and it's the first end-to-end medical imaging segmentation framework. When I discovered your work, I was extremely excited. I believe I have finally found a highly valuable reference paper. I am very eager to discuss this research with you and learn from you.
